lib/samples/add_tiled_layer_as_basemap/README.md

```diff
@@ -0,0 +1,43 @@
+# Add tiled layer as basemap
+
+Load an offline copy of a tiled map service as a basemap.
+
+![Image of add tiled layer as basemap](add_tiled_layer_as_basemap.png)
+
+## Use case
+
+Constructing an ArcGISTiledLayer from a local tile package (.tpk & .tpkx), allows you to use that basemap tiled service when the application is offline. Dividing a raster into tiles allows the map to provide relevant tiles and level of detail to the user when panning and zooming. For example, when working in an environment that has no connectivity, this could provide access to a map for navigating your surroundings.
+
+## How to use the sample
+
+Launch the app to view the "San Francisco offline tile package" as the basemap.
+
+## How it works
+
+1. Create a `TileCache`, specifying the path to the local tile package.
+2. Create an `ArcGISTiledLayer` with the tile cache.
+3. Create a `Basemap` with the tiled layer.
+4. Create an `ArcGISMap` with the basemap and set it on the `ArcGISMapViewController`.
```

Comment:
```suggestion
4. Create an `ArcGISMap` with the basemap and set it on the `ArcGISMapView`.
```
Again from an ArcGIS POV it feels like this should reference ArcGISMapView. If we go with referencing the controller we should add this to the note of things to go back to.